dts/dts.cmake

```diff
@@ -57,6 +57,7 @@ if(CONFIG_HAS_DTS)
     ${DTC_INCLUDE_FLAG_FOR_SOURCE}  # include the DTS source
     ${DTC_INCLUDE_FLAG_FOR_OVERLAY} # Possibly include an overlay after the source
     -I${PROJECT_SOURCE_DIR}/dts/common
+    -I${PROJECT_SOURCE_DIR}/dts/bindings/interrupt-controller
```

Comment:
this shouldn't be required.

The include should be like: #include <dt-bindings/i2c/i2c.h>